test/items/dire-hit.test.ts

```diff
@@ -69,7 +69,7 @@ describe("Items - Dire Hit", () => {
       "SelectModifierPhase",
       UiMode.MODIFIER_SELECT,
       () => {
-        const handler = game.scene.ui.getHandler() as ModifierSelectUiHandler;
+        const handler = game.scene.ui.getCurrentHandler() as ModifierSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = game.scene.ui.getCurrentHandler<ModifierSelectUiHandler>();
```